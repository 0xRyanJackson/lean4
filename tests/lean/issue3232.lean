inductive foo {n : Nat} : Prop

-- Check that the error message will print the types with implicit arguments shownx
example (h : @foo 42) : @foo 23 := by
  apply h
