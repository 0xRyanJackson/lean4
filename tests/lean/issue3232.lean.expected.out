issue3232.lean:5:2-5:9: error: tactic 'apply' failed, failed to unify
  @foo 42
with
  @foo 23
h : foo
‚ä¢ foo
